ontology URL

Ahmedreyadh1985sh opened the pull request.

dgarijo commented:
@Ahmedreyadh1985sh you are creating a `GVTPDonto` folder with no redirection. Can you please delete that one?

Ahmedreyadh1985sh commented:
Dear sir what do you mean without redirect I added files ( .htaccess and
README) to the folder ( GVTPDonto) and I want to redirect the address (
https://w3id.org/GVTPDOnto or  https://w3id.org/GVTPDonto) to (
https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html).
Note that the address (https://w3id.org/GVTPDOnto) was previously reserved,
and I updated the ontology, and I would like it to refer to it.
What is the correct procedure? with respect

‫في الخميس، 10 أغسطس 2023 في 12:09 م تمت كتابة ما يلي بواسطة ‪Daniel
Garijo‬‏ ***@***.***‬‏>:‬

> @Ahmedreyadh1985sh <https://github.com/Ahmedreyadh1985sh> you are
> creating a GVTPDonto folder with no redirection. Can you please delete
> that one?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/perma-id/w3id.org/pull/3527#issuecomment-1672848986>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A36KOWOROWGFNLGA3Y3WJ5TXUSQOHANCNFSM6AAAAAA3KJKCXU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Ahmedreyadh1985sh commented:
i want to update (.htaccess) file
